wikimedia/wikipedia-ios
# Native Editor - add find

**Phabricator:** 
https://phabricator.wikimedia.org/T348091

### Notes
This adds find functionality to the native editor. I will put up the replace pieces in a followup PR. 

Note:
I notice that iOS 17 / TextKit 2 doesn't always reliably scroll to the correct selected match. I haven't found a quick fix for this yet, but if it becomes too big of a problem we can easily switch to the TextKit 1 implementation.

### Test Steps
1. Launch app in Staging scheme, navigate to the editor.
2. Tap the find formatting button
3.  Type in an existing word in the find text field. Confirm you see find results highlighted and counts displayed. 
4. Tap the previous and next buttons, confirm you can traverse matches.
